src/server/templates/page/PageHome.jsx

```diff
@@ -8,7 +8,6 @@ export const PageHome = () => {
             head={
                 <>
                     <title>LuXE Demo</title>
-                    <meta name="description" content="luXE: demonstration website created by Jahia. Jahia the best user experience"/>
```

Comment:
@herve Removing as it conflicts with the new SEO fields. just want to check if you have thoughts on it (as it seems you've added it in), or if ok for now.